.github/workflows/build.yaml

```diff
@@ -32,10 +32,31 @@ jobs:
       id: go  # XXX: what is this for?
     - name: Install govendor
       run: go get -u github.com/kardianos/govendor
-    # This dominates the duration of the test. Can we get a smaller set that is
-    # enough to build embedded C parts?
-    - name: Get C dependencies
-      run: sudo apt-get build-dep -y ./src/github.com/snapcore/snapd
+    - name: Cache subset of C dependencies
+      id: cache-deb-downloads
+      uses: actions/cache@v1
+      with:
+        path: /tmp/debian-packages/
+        key: debian-packages-{{ hashFiles('**/debian/control') }}
+    - name: Download a subset of C dependencies
+      if: steps.cache-deb-downloads.outputs.cache-hit != 'true'
+      run: |
+          mkdir /tmp/debian-packages
+          cd /tmp/debian-packages
+          sudo apt update
```

Comment:
I think we should go with just "sudo apt update" for now. Then we can simply cache the content of /var/cache/apt/archives after we did a "apt build-dep -y ./..."

Reply:
I tried that but it's failing because of apt lock files and the fact that this cache is enormous.